server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -423,20 +424,28 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
 	public handler(rawMessage: IQueuedMessage) {
 		// In cases where we are reprocessing messages we have already checkpointed exit early
 		if (rawMessage.offset <= this.logOffset) {
-			Lumberjack.info(
-				`rawMessage.offset: ${rawMessage.offset} <= this.logOffset: ${this.logOffset}`,
-				getLumberBaseProperties(this.documentId, this.tenantId),
-			);
+			const reprocessOpsMetric = Lumberjack.newLumberMetric(LumberEventName.ReprocessOps);
+			reprocessOpsMetric.setProperties({
+				...getLumberBaseProperties(this.documentId, this.tenantId),
+				kafkaMessageOffset: rawMessage.offset,
+				databaseLastOffset: this.logOffset,
+			});
```

Comment:
Do we want to try the following code block, catch any errors and conclude/complete the metric with `reprocessOpsMetric.error(...)`?

Reply:
Added try/catch block to catch any errors